docker-compose.yml

```
@@ -21,6 +21,24 @@ services:
       - SESSION_SECRET=somesecretvalue
       - INGRESS_URL=http://localhost:3000
 
+  gotenberg:
+    image: gotenberg/gotenberg:7.7.0
+    networks:
+      - hmpps
```

hmpps network is not defined, so docker-compose up fails, suggested change:-
```suggestion
      - laa
```